rpm/patchmanager.spec

```diff
@@ -61,6 +61,8 @@ ln -s ../lipstick-patchmanager.service %{buildroot}/%{_userunitdir}/lipstick.ser
 
 mkdir -p %{buildroot}%{_datadir}/%{name}/patches
 
+install -m 644 -D src/etc/whitelist-common-patchmanager.local %{buildroot}%{_sysconfdir}/firejail/whitelist-common-patchmanager.local
```

Comment:
I use plain install here as I am not familiar how a .pro file would do this.

For sake of consistency it would probably be good to do this in a .pro file like everything else does.